webdriver-spec.html

```diff
@@ -6878,23 +6878,24 @@
 <section class=appendix>
 <h2>Security</h2>
 
-<p>A user agent may rely on a command-line flag
- or a configuration option to test whether to enable WebDriver.
- Another option is to make the user agent initiate or confirm
- the connection through a privileged content document or control widget,
- in case the user agent does not directly implement the HTTP endpoints.
-
-<p>To prevent arbitrary machines from connecting and creating <a>sessions</a>,
- only connections from loopback devices are allowed by default.
- <a>Remote ends</a> do however support a scenario
- where the <a>local end</a> runs on a different machine
- from the <a>remote end</a>.
-
-<p>The <a>remote end</a> must include a configuration option
- to limit the accepted IP range allowed to connect and make requests.
- The default setting for this should be
- to limit connections to the IPv4 localhost CIDR range <code>127.0.0.0/8</code>
- and the IPv6 localhost address <code>::1</code>. [[!RFC4632]]
+<p>The following suggestions are made to implementors to increase the
+ security of users of WebDriver:
+
+ <ul>
+  <li>Provide a command-line flag or configuration option to enable
+   WebDriver.
+  <li>Only listen on loopback devices for requests for
+   new <a>sessions</a>.
+  <li>Provide a configuration option to limit the accepted IP range
+   allowed to connect and make requests, such as the IPv4 localhost
+   CIDR range <code>127.0.0.0/8</code> and the IPv6 localhost
+   address <code>::1</code>. [[!RFC4632]]
```

Comment:
Seems like this should be default but there should also be the ability to designate an address and and to do so over https and or http2 with TLS. 
This is common practice with databases. 
Ultimately lots of folks will want to use or implement a grid of systems. 

> On Sep 5, 2016, at 8:20 PM, David Burns <notifications@github.com> wrote:
> 
> In webdriver-spec.html:
> 
> > - to limit the accepted IP range allowed to connect and make requests.
> > - The default setting for this should be
> > - to limit connections to the IPv4 localhost CIDR range <code>127.0.0.0/8</code>
> > - and the IPv6 localhost address <code>::1</code>. [[!RFC4632]]
> > +<p>The following suggestions are made to implementors to increase the
> > + security of users of WebDriver:
> > +
> > + <ul>
> > +  <li>Provide a command-line flag or configuration option to enable
> > +   WebDriver.
> > +  <li>Only listen on loopback devices for requests for
> > +   new <a>sessions</a>.
> > +  <li>Provide a configuration option to limit the accepted IP range
> > +   allowed to connect and make requests, such as the IPv4 localhost
> > +   CIDR range <code>127.0.0.0/8</code> and the IPv6 localhost
> > +   address <code>::1</code>. [[!RFC4632]]
> We should include what the default is here.
> 
> ―
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>